posts/2024-05-10_new_admiral_ex.../new_admiral_extension_packages_admiralpeds_coming_soon.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "New {admiral} extension packages: {admiralpeds} & more coming soon!"
+author:
+  - name: Fanny Gautier
+  - name: Lina Patil
+description: "We are very excited to announce that new extension packages are expected very soon in the {admiral} family: {admiralpeds}, a pediatrics extension, and another extension (name TBC) for obesity in clinical trials"
+date: "2024-05-10"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Metadata, ADaM, Community]
+# feel free to change the image
+image: "coming_soon.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-05-10_new_admiral_ex..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+[![](admiralpeds.png){fig-align="center" width="220"}](https://pharmaverse.github.io/admiralpeds/index.html)
+
+📢 In the pharmaverse, we are not just adopting new tools, but also fostering a culture of collaboration, innovation and a commitment to continuous learning. We are thus thrilled to update you on the latest developments from our {admiral} family team: [`{admiralpeds} 0.1`](https://pharmaverse.github.io/admiralpeds/index.html) is __coming soon__ [on CRAN](https://cran.r-project.org)!
+
+`{admiralpeds}` is a pediatrics extension package for  `{admiral}`. Its first release is planned by July 2024.
+
+# Collaboration
+
+7 companies (Roche, Pfizer, Amgen, BMS, J&J, Novartis, and Cytel) and David Freedman, a passionate consultant retired from CDC, are working together to create this extension 🤝.
```

Comment:
```suggestion
4 companies (Roche, Pfizer, GSK, and Cytel), a variety of other community contributors, and David Freedman, a passionate consultant retired from CDC, are working together to create this extension 🤝.
```

Reply:
@rossfarrugia GSK was asked to remove from the list by @bms63 in one of the earlier comments. Could you please confirm to add it back? And, is there any reason the remove Amgen, BMS, J&J and Novartis?

Reply:
thanks Lina! good point on GSK - i'll also update that here https://github.com/pharmaverse/admiralpeds/pull/22. None of Amgen, BMS, J&J and Novartis co-develop admiralpeds with us. they all support other package extensions.